frameworks/digital-outcomes-and-specialists-2/manifests/declaration.yml

```diff
@@ -43,7 +43,8 @@
     Check that your organisation complies with [Regulation 57(1) of the Public
     Contracts Regulations 2015](http://www.legislation.gov.uk/uksi/2015/102/regulation/57/made).
   questions:
-    - conspiracyCorruptionBribery
+    - conspiracy
+    - CorruptionBribery
```

Comment:
We should keep this and the filename as `corruptionBribery` to avoid any issues when switching between case-insensitive and case-sensitive file systems